test/RPCSession.cpp

```diff
@@ -139,9 +139,16 @@ string IPCSocket::sendRequest(string const& _req)
 
 RPCSession& RPCSession::instance(const string& _path)
 {
-	static RPCSession session(_path);
-	BOOST_REQUIRE_EQUAL(session.m_ipcSocket.path(), _path);
-	return session;
+	try
+	{
+		static RPCSession session(_path);
```

Comment:
@ekpyron This line throws an `std::length_error` if `aleth` is not running due to an initialization error of `IPCSocket` . Problem is that the connection to the socket fails, but `isoltest` does not recognize the failure because `BOOST_FAIL` is used and nothing throws.